pkgs/development/python-modules/rabbitpy/default.nix

```diff
@@ -18,6 +18,11 @@ buildPythonPackage rec {
   buildInputs = [ mock nose ];
```

Comment:
```suggestion
  checkInputs = [ mock nose ];
```

Reply:
I actually don't see any tests in the tarball.

Can you switch to using the github source?
(also add a comment that there's no tests in the pypi tarball)

Reply:
Did change to the github source.

Reply:
> Did change to the github source.

:confused: , I don't see it in the diff?

Reply:
@worldofpeace Sorry, did comment a little prematurely.